File: CloudFlow/FrontEnd-App/src/app/shared/model/content/get-calendars-events-request.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class GetCalendarsEventsRequestModel {

	@JsonProperty("calendarIds", [Number])
	public calendarIds: number[] = null;

	@JsonProperty("startDate", JsonDateConverter)
	public startDate: Date = null;

	@JsonProperty("endDate", JsonDateConverter)
	public endDate: Date = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/GetCalendarsEventsRequestModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content;

import java.util.Date;
import java.util.List;

public class GetCalendarsEventsRequestModel {
	
	private List<Long> calendarIds;
	private Date startDate;
	private Date endDate;
	
	public List<Long> getCalendarIds() {
		return calendarIds;
	}
	public void setCalendarIds(List<Long> calendarIds) {
		this.calendarIds = calendarIds;
	}
	public Date getStartDate() {
		return startDate;
	}
	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}
	public Date getEndDate() {
		return endDate;
	}
	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}
}
